### ISSUE TITLE
Burger Menu can not navigate to page or dismiss

### ISSUE THREAD

**Issue by daKmoR** [GoogleChrome/web.dev#7749]
**Describe the bug**
You can not navigate on mobile via the burger menu 😅

**To Reproduce**
1. Go to https://web.dev/ on a small screen/mobile
2. Click on burger menu
3. Can not navigate to a page
4. Can not close overlay

**Expected behavior**
- Burger Menu allows me to navigate to different pages
- I can close the Burger menu by clicking outside or on the "x"

**Version Info**
- OS: macOS Monterey (v12.3)
- Browser: Chrome: Version 100.0.4896.127 (Official Build) (x86_64)

**Video**

https://user-images.githubusercontent.com/24378/164080292-05c44c3a-f378-489e-9168-9549acabd268.mp4

**Comment by jeffposnick** [comment]
Thanks for reporting this, @daKmoR!

I feel like it may have to do with the native browser implementation of `inert` varying from what the [`wicg-inert` polyfill](https://github.com/WICG/inert) provides, but `inert` didn't ship in Chrome stable until [102](https://chromestatus.com/feature/5703266176335872). If you were seeing it earlier, perhaps it's because you had an experimental `chrome://flags` enabled?

In any case, I can reproduce the problems on both Chrome and Safari (which also supports `inert` natively) right now, but not in Firefox (which doesn't have support, and is therefore still using the polyfill).

**Comment by daKmoR** [comment]
yes - I definitely had the "Experimental Web Platform features" flag enabled

seems a reasonable cause - nice detective work 👍
